magento/adobe-stock-integration
# Loader stays over the media gallery if it was opened second time from…

### Description (*)

Depends On https://github.com/magento/magento2/pull/27566
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#935: Loader stays over the media gallery if it was opened second time from different control on the same page
2. ...

### Manual testing scenarios (*)
 Preconditions (*)
1. Install Magento and Adobe Stock Integration [See documentation](https://github.com/magento/adobe-stock-integration/wiki/Installation-Guide)

 Steps to reproduce (*)
1. Login to Admin panel
2. Open Media Gallery (i.e. Catalog -> Category -> expand Content -> Select from Gallery button)
3. Close the Media Gallery
4. Open the Media Gallery using different entry point on the same page (i.e. WYSIWYG)

Expected result (*)
Media Gallery is loaded and loader disappears
